tools/pkglint/lib/rules.ts

```diff
@@ -1457,6 +1457,43 @@ export class JestSetup extends ValidationRule {
   }
 }
 
+/**
+ * Most packages in v2 must be private. Carry an allowlist so we don't
+ * accidentally publish private packages.
+ */
+export class V2PublicPackages extends ValidationRule {
+  public readonly name = 'v2/public-package';
+  private readonly allowlist = ['aws-cdk-lib', 'cdk', 'aws-cdk', 'awslint'];
```

Comment:
feels like we are missing some stuff here.... `decdk`, `cdk-dasm` come to mind and more. how did we compile this list?

Reply:
this is not the final list. it's just a start. we'll adjust as we go.